### PR TITLE
Add withdraw method to cli

### PR DESCRIPTION
**Pull request by Krigpl**

### CODE REVIEW

File: golem/interface/client/account.py [hunk 1]
```diff
@@ -4,13 +4,17 @@
...
 from golem.core.deferred import sync_wait
-from golem.interface.command import command, group
+from golem.interface.command import Argument, command, group
 
 
 @group(help="Manage account")
 class Account:
     client = None  # type: 'golem.rpc.session.Client'
 
+    amount_arg = Argument('amount', help='Amount to withdraw, eg 1.45')
```
Comment (etam) [inline golem/interface/client/account.py:15]: In what units is this amount? I think that it's not '1.45', but rather 1.45*10^18. Please check and update the help message.

File: golem/interface/client/account.py [hunk 2]
```diff
@@ -70,6 +75,17 @@ def unlock(self) -> str:  # pylint: disable=no-self-use
...
         return "Account unlock success"
 
+    @command(
+        arguments=(amount_arg, address_arg, currency_arg),
+        help="Withdraw GNT/ETH")
+    def withdraw(
+            self,
+            amount,
+            destination,
+            currency) -> str:  # pylint: disable=no-self-use
```
Comment (Wiezzel) [inline golem/interface/client/account.py]: Why hack pylint rather than make this method static?

File: golem/interface/client/account.py (hunk 1 above)
Comment (Krigpl) [reply golem/interface/client/account.py:15]: youdidntseeanythingpinguin
Yeah, I realized immediately after pushing, it's fixed now.

Comment (etam) [reply golem/interface/client/account.py:15]: Ah, you make `int(Decimal(amount) * denoms.ether)`

File: golem/interface/client/account.py (hunk 2 above)
Comment (Krigpl) [reply golem/interface/client/account.py]: https://github.com/golemfactory/golem/pull/2463/files
Python's stuff I guess

**Review by etam — APPROVED** [review]
ok

**Review by mfranciszkiewicz — APPROVED** [review]

File: golem/interface/client/account.py (hunk 2 above)
Comment (Wiezzel) [reply golem/interface/client/account.py]: You can as well use `self.client` instead of `Account.client` to avoid warning. :stuck_out_tongue_winking_eye:

Comment (mfranciszkiewicz) [reply golem/interface/client/account.py]: `# pylint: disable=no-self-use` should be at the first line of function definition

**Comment by etam** [comment]
Houston! We have a problem:
```
./golemcli.py account withdraw 99983333333333333332 0xMyEthereumAddress GNT
[10:47:19.680897] ERROR: Exception: WAMP message serialization error: huge unsigned int
```

**Comment by etam** [comment]
Houston! We have a problem:
```
./golemcli.py account withdraw 99.983333333333333332 0xMyEthereumAddress GNT
[10:49:47.345479] ERROR: Exception: WAMP message serialization error: huge unsigned int
```

**Comment by Wiezzel** [comment]
@etam Googling for this error message returns some interesting results:
https://github.com/golemfactory/golem/issues/1632
https://github.com/golemfactory/golem/issues/1017
https://github.com/golemfactory/golem/issues/864

**Comment by mfranciszkiewicz** [comment]
Cast and send as a str.

**Comment by maaktweluit** [comment]
Y'all have too much GNT, ints work for me ;)

```
(.venv) ubuntu@ip-172...:~/src/golem$ python golemcli.py account withdraw 1.683333333333333342 0x... GNT
- '0x...'
```

**Pull request merged**